app/Models/Settings/Accounts/GeneralLedgerAccounts.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Models\Settings\Accounts;
+
+use Auth;
+use App\Models\School;
+use App\Scopes\SchoolScope;
+use Illuminate\Database\Eloquent\Model;
+
+class GeneralLedgerAccounts extends Model
```

Comment:
i think you should make this class name singular instead of plural to avoid future problems

Reply:
Will consider it given the priority now.